test/column/formatter.test.tsx

```diff
@@ -114,3 +114,41 @@ describe('Custom formatter component', () => {
     });
   });
 });
+
+describe('formatter focus', () => {
```

Comment:
Doesn't matter too much, but isn't this about `cell` focus rather than `formatter` focus?